plugins/server/org.jetbrains/kotlinx-rpc/3.0/manifest.ktor.yaml

```diff
@@ -0,0 +1,39 @@
+name: kotlinx.rpc
+description: Adds remote procedure call (RPC) routing
+vcsLink: https://github.com/Kotlin/kotlinx-rpc
+license: Apache 2.0
+category: Frameworks
+maven:
+  disabled: true
+# TODO waiting for fixes to enable amper support
```

Comment:
please log issue for this as well

Reply:
There is one now, I'll add it to the TODO